models.py

```diff
@@ -45,14 +46,33 @@ class Jurisdiction(db.Model):
     batches = relationship('Batch', backref='jurisdiction', passive_deletes=True)    
     audit_boards = relationship('AuditBoard', backref='jurisdiction', passive_deletes=True)
     contests = relationship('TargetedContestJurisdiction', backref='jurisdiction', passive_deletes=True)
+    admin_accesses = relationship('UserJurisdictionAccess', backref='jurisdiction', passive_deletes=True)
 
 # users that log in
 class User(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
-    email = db.Column(db.String(150), unique=True, nullable=False)
+    id = db.Column(db.String(200), primary_key=True)
+    external_id = db.Column(db.String(200), unique=True, nullable=False)
+
+    election_roles = relationship('UserElectionRole', backref='user', passive_deletes=True)
+    jurisdiction_accesses = relationship('UserJurisdictionAccess', backref='user', passive_deletes=True)
+
+class UserRole(Enum):
+    AuditAdmin = 'AuditAdmin'
+    JurisdictionAdmin = 'JurisdictionAdmin'
+    AuditBoardMember = 'AuditBoardMember'
```

Comment:
I wonder if we should represent audit board members the same way here, given that there is never any normalization across elections, and we don't actually ever have identifiers for audit board members other than the audit board ID we already generate in the other table.

If we limited this explicit data modeling to Audit and Jurisdiction admins, then we could have the roles table referencing an election, a role, and optionally a jurisdiction. One less table, easier to work with, I think?